proposals/xxxx-work-graphs.md

```diff
@@ -0,0 +1,535 @@
+<!-- {% raw %} -->
+
+# Work Graphs
+
+* Proposal: [NNNN](NNNN-work-graphs.md)
+* Author(s): [Chris Bieneman](https://github.com/llvm-beanz)
+* Sponsor: [Greg Roth](https://github.com/pow2clk), [Tex
+  Riddell](https://github.com/tex3d)
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8
```

Comment:
Shouldn't we update these fields, since really aren't going through approval process for an already-shipped feature.
Also, can we assign a number to this now?  I'd like for the mesh nodes spec to have a number that's higher than this one, since it extends it 😄.

Reply:
Renumbered and classified.